testsuite/tests/singlecluster/observability/conftest.py

```diff
@@ -0,0 +1,75 @@
+"""Conftest for observability tests"""
+
+import time
+
+import backoff
+import pytest
+from openshift_client import selector
+
+from testsuite.kubernetes.monitoring.pod_monitor import PodMonitor
+from testsuite.kubernetes.monitoring.service_monitor import ServiceMonitor
+
+
+@pytest.fixture(scope="module")
+def commit():
+    """Override the commit fixture to do nothing"""
+    return None
+
+
+@pytest.fixture(scope="module", autouse=True)
+def no_observability_metrics_before_enable(prometheus):
```

Comment:
how much time it take for targets to get removed after monitors are removed? Are there any metric properties that can indicate when they will be removed?